installing/installing_aws/installing-aws-localzone.adoc

```diff
@@ -3,87 +3,61 @@
 = Installing a cluster on AWS with worker nodes on AWS Local Zones
 include::_attributes/common-attributes.adoc[]
 :context: installing-aws-localzone
+:zone-type: Local Zones
```

Comment:
I see the logic. I reduced the amount of high-level attributes because of the nature of others using these files and I wanted to avoid a situation where attributes get mixed up, so I ensured "local-zone" and "wavelength-zone" were hardcoded. If I see major issues further on its the comment, I'll check the extra attributes.